frb_codegen/src/main.rs

```diff
@@ -26,3 +26,195 @@ fn main() -> anyhow::Result<()> {
     info!("Now go and use it :)");
     Ok(())
 }
+
+mod tests {
+    use std::fs;
+
+    use lazy_static::lazy_static;
+    use lib_flutter_rust_bridge_codegen::init_logger;
+
+    lazy_static! {
+        static ref LOGGER: () = init_logger(".", true).unwrap();
+    }
+
+    // VS Code runs in frb_codegen with "Run test" and flutter_rust_bridge with "Debug test" >_>
+    #[allow(dead_code)]
```

Comment:
uber-nit: is it possible to diable allow_deadcode

Reply:
fixed